testutils/memory.go

```diff
@@ -0,0 +1,120 @@
+package testutils
+
+import (
+	"github.com/src-d/metadata-retrieval/github/graphql"
+	"gopkg.in/src-d/go-log.v1"
+)
+
+// Memory implements the storer interface
+type Memory struct {
+	Organization *graphql.Organization
+	Repository   *graphql.RepositoryFields
+	Topics       []string
+	Users        []*graphql.UserExtended
+	Prs          []*graphql.PullRequest
+	PrComments   []*graphql.IssueComment
```

Comment:
nitpick: rename to `PRs` and `PrComments`, [reason](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)